src/main/ui/app/route/Show.jsx

```diff
@@ -154,53 +191,63 @@ export default class Show extends Component {
-                                    <th className="col-md-2 col-sm-2 col-xs-2">Release Date</th>
-                                    <th className="col-md-3 col-sm-3 col-xs-3">Platform Requirement<OverlayTrigger
-                                            placement="right" overlay={requirementmeaning}>
-                                        <Glyphicon glyph="glyphicon glyphicon-question-sign"/>
-                                    </OverlayTrigger></th>
-                                    <th className="col-md-5 col-sm-5 col-xs-5">Other requirements</th>
-                                    <th className="col-md-1 col-sm-1 col-xs-1">Download</th>
-                                </tr>
-                                </thead>
-                                <tbody>
-                                {addon.versions.map(v => {
-                                    let className = v.version === highlightVersion ? "highlight" : "";
-                                    return (
-                                            <tr className={className}>
-                                                <td>
-                                                    {v.version}
-                                                </td>
-                                                <td>
-                                                    {this.formatDateTime(v.releaseDatetime)}
-                                                </td>
-                                                <td>
-                                                    {v.requireOpenmrsVersion}
-                                                </td>
-                                                <td>
-                                                    {this.formatRequiredModules(v)}
-                                                </td>
-                                                <td>
-                                                    <Button bsSize="small"
-                                                            href={v.renameTo ? `/api/v1/addon/${addon.uid}/${v.version}/download` : v.downloadUri}>
-                                                        <i className="fa fa-download"></i>
-                                                        Download
-                                                    </Button>
-                                                </td>
-                                            </tr>
-                                    )
-                                })}
+                        <div className="col-md-2 col-sm-2 col-xs-2 pull-right">
+                            <Button href={version.downloadUri} bsStyle="primary" bsSize="large">
+                                <span className="buttonstyleheading">Download
+                                </span><br/>
+                                <span className="buttonstylesubtext">
+                                Version : {version.version}
```

Comment:
I would expect us to say either "Latest Version (version number)" or "Latest Supported Version (version number)"